src/Library/Plugin/UpdateData.php

```diff
@@ -194,14 +194,16 @@ public function getResponseData() {
 	 * @return Response
 	 */
 	public function fetchResponseData( $force = false ) {
+		global $wp_filter;
 		$is_timely_updates  = apply_filters( 'boldgrid_backup_is_timely_updates', false );
 		$plugin_information = array();
 		$delay_time         = $force ? 0 : 3;
 		$delayFetchingData  = ( $this->getAgeOfTransient() < $delay_time );
 		if ( $is_timely_updates && ! $delayFetchingData ) {
+			\remove_all_filters( 'plugins_api' );
```

Comment:
(1) Removing a filter like this seems like a big thing. Is this going to cause certain plugins not to be able to update themselves? What exactly is the bug that this is fixing?

---

> If other plugins stop working because of this, thats their fault and they should make better filters.

Sounds pretty harsh lol. Take a breath. Who's doing what wrong, and what's happening because of it?

Reply:
Can you add some comments to your code about these 2 items, for future reference, and then I think we can approve.

Reply:
Yeah. That I can do